packages/commonwealth/client/scripts/controllers/app/login.ts

```diff
@@ -549,6 +551,33 @@ export async function handleSocialLoginCallback({
         : app.config.chains.getById(app.activeChainId());
       await updateActiveAddresses({ chain: c });
     }
+
+    // TODO: behind a feature flag
```

Comment:
some feature flag toggle is missing here or?

Reply:
oh yes, its missing, thanks for the spot. Will update

Reply:
Fixed.